package-lock.json

```diff
@@ -49230,4 +49230,4 @@
             }
         }
     }
-}
```

Comment:
not sure what this change is for :thinking:

Reply:
It deleted the EOF and added a new curley bracket so there is no difference to the dev branch package-lock.json.